pkg/metrics/metrics.go

```diff
@@ -212,3 +208,16 @@ func EmitCSVMetric(oldCSV *olmv1alpha1.ClusterServiceVersion, newCSV *olmv1alpha
 		csvAbnormal.WithLabelValues(newCSV.Namespace, newCSV.Name, newCSV.Spec.Version.String(), string(newCSV.Status.Phase), string(newCSV.Status.Reason)).Set(1)
 	}
 }
+
+func EmitSubsMetrics(sub *olmv1alpha1.Subscription) {
+	// sub.Spec is not a required field.
+	if sub.Spec == nil {
+		return
+	}
```

Comment:
As far as I can tell, this is no longer true. I tried applying the following files:
```
$ cat empty.sub.cr.yaml 
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: my-operator-subsription
  namespace: default

$ oc apply -f empty.sub.cr.yaml 
The Subscription "my-operator-subsription" is invalid: spec: Required value
```

&

```
$ cat empty.sub.cr.yaml 
apiVersion: operators.coreos.com/v1alpha1
kind: Subscription
metadata:
  name: my-operator-subsription
  namespace: default
spec: {}

$ oc apply -f empty.sub.cr.yaml 
The Subscription "my-operator-subsription" is invalid: 
* spec.name: Required value
* spec.source: Required value
* spec.sourceNamespace: Required value
```